policies/documentation-policy.md

```diff
@@ -0,0 +1,91 @@
+# OpenSSL documentation policy
+
+This document describes the code documentation and commenting
+requirements for the OpenSSL project.
```

Comment:
As we certainly do not fully adhere to this policy with the current code base we should state that the policy in full applies only to new additions to the code. I.E. we do not require authors of trivial bugfix PRs to add a documentation of the internal function being fixed if it is missing and we do not plan/we are not required to make the existing code base to comply with the policy ASAP. The compliance will be just incrementally improved.